--- silc-toolkit-1.1.10/lib/Makefile.am.orig	2009-08-07 13:51:26.000000000 +0200
+++ silc-toolkit-1.1.10/lib/Makefile.am	2013-01-30 18:19:15.291151418 +0100
@@ -82,14 +82,14 @@
 LIBSILC_AGE=@LIBSILC_AGE@
 
 libsilc.a:
-	find $(SILCLIB_DIRS) -type f -name *.lo | xargs	$(LIBTOOL) --mode=link $(CC) $(CFLAGS) $(LDFLAGS) $(SILC_LINK_LIBS)	$(LIBTOOL_SILC_VERSION)	$(LIBTOOL_OPTS) -o libsilc.la
+	find $(SILCLIB_DIRS) -type f -name *.lo | xargs	$(LIBTOOL) --mode=link $(CC) $(CFLAGS) $(LDFLAGS) $(SILC_LINK_LIBS)	$(LIBTOOL_SILC_VERSION)	$(LIBTOOL_OPTS) -o libsilc.la -lpthread -ldl
 
 LIBSILCCLIENT_CURRENT=@LIBSILCCLIENT_CURRENT@
 LIBSILCCLIENT_REVISION=@LIBSILCCLIENT_REVISION@
 LIBSILCCLIENT_AGE=@LIBSILCCLIENT_AGE@
 
-libsilcclient.a:
-	find $(SILCCLIENTLIB_DIRS) -type f -name *.lo | xargs	$(LIBTOOL) --mode=link $(CC) $(CFLAGS) $(LDFLAGS)	$(SILCCLIENT_LINK_LIBS) $(LIBTOOL_SILCCLIENT_VERSION)	$(LIBTOOL_OPTS) -o libsilcclient.la
+libsilcclient.a: libsilc.a
+	find $(SILCCLIENTLIB_DIRS) -type f -name *.lo | xargs	$(LIBTOOL) --mode=link $(CC) $(CFLAGS) $(LDFLAGS)	$(SILCCLIENT_LINK_LIBS) $(LIBTOOL_SILCCLIENT_VERSION)	$(LIBTOOL_OPTS) -o libsilcclient.la libsilc.la
 
 
 pkgconfigdir = $(libdir)/pkgconfig
